### DISCUSSION TITLE
有没有必要去掉 外部访问地址 参数

### DISCUSSION THREAD

**Discussion by qq547176052** [halo-dev/halo#3093] [General]
![image](https://user-images.githubusercontent.com/26636842/210149041-d18ffe3a-bcd8-48dd-81c8-310cf21984a9.png)
这个参数去掉

**Comment by longtengkeji** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
